Tasks/UserInterface/03a_List.md

```diff
@@ -0,0 +1,119 @@
+[!!Email settings](../UserInterface/02d_EmailSettings.md)
+[!!Manage the events](../Operation/01_ManageEvents.md)
+
+# List (Events)
+
+*Tasks > Events > Tab LIST*
+
+![List of events](../../Assets/Screenshots/Tasks/Events/ListEvents.png "[List of events]")
+
+**List of events**
+
+The list displays all events created in the system. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+The following functions are available for the editing toolbar:
+
+- [EDIT]  
+    Click this button to edit an event. This button is only displayed if the checkbox of an event is selected. The [Edit event](#edit-event) view is displayed.
+
+- [DELETE]  
+    Click this button to delete an event. This button is displayed if the checkbox of at least one event is selected.
+
+The following functions and fields are available in this view:
+
+- *Name*  
+    Event name.
+
+- *Plugin*  
+    Module or plugin that owns the event. 
+    
+    > [Info] Most events are created automatically by the system when installing a module or plugin. If you create an event manually in the *Tasks* module, **Manually created** is displayed.
+
+- *Responsible user*  
+    User to whom the event is assigned. If the event is not assigned to any user, **Unassigned** is displayed.
+
+- *Responsible user group*  
+    User group to whom the event is assigned. If the event is not assigned to any user group, **Unassigned** is displayed.
+
+- *Obsolete/Active*  
+    Indication whether the event is **Active** or **Obsolete**.
+
+- *Identifier*  
+    Event identifier. The event identifier is similar to a key and therefore must be system-wide unique.
+
+- *ID*  
+    Event identification number. The ID number is automatically assigned by the system.
+
+
+
+## Create event
+
+*Tasks > Events > Tab LIST > Button Add*
+
+![Create event](../../Assets/Screenshots/Tasks/Events/CreateEvent.png "[Create event]")
+
+The following functions and fields are available in this view:
+
+- *"Language name"*  
+    Click the drop-down list and select the language in which the *Name (Language)* is displayed. By default, the following options are available:
+
+    - **English (United States)**
+    - **Deutsch (Deutschland)**
+
+- *Name*  
+    Enter a name for the event.
+
+- *Identifier*  
+    Event identifier. The event identifier is similar to a key and therefore must be system-wide unique.
```

Comment:
Enter an identifier for the event. It is similar to a key and and must be unique.

Oben Enter a name. Darum müsste hier auch Enter an identifier stehen